scripts/image_functions.sh

```diff
@@ -2,13 +2,14 @@
 export PYTHONIOENCODING
 
 function cleanup() {
-	find . -name "${IMAGE_NAME}" -type d -exec rm -rf '{}' \;
-	# delete any orphaned temporary images
+	find . -name "${IMAGE_NAME}*.d" -type d -exec rm -rf {} +
+    find . -name "${IMAGE_NAME}*.qcow2" -type f -exec rm -f {} +
+	# delete any orphaned temporary images using image_id
 	for id in $(glance image-list | grep "$TMP_IMAGE_NAME" | awk '{print $2}')
 	do
 		glance image-delete $id &>/dev/null || true
 	done
-	# delete any orphaned test instance
+	# delete any orphaned test instance using instance_id
```

Comment:
instane_id ? or instance_name?

Reply:
Instance_id, testing a new commit of this branch

Reply:
Woosp I meant on row 13. 'for id in $(nova list | grep "$TEST_INSTANCE_NAME" | awk '{print $2}')'